spec/swagger/v1/schemas/recipe_schema.rb

```diff
@@ -0,0 +1,65 @@
+RECIPE_SCHEMA = {
+  type: :object,
+  properties: {
+    user_id: { type: :integer, example: 666, 'x-nullable': true },
+    name: { type: :string, example: 'Some name', 'x-nullable': true },
+    portions: { type: :integer, example: 666, 'x-nullable': true },
+    instructions: { type: :string, example: 'Some instructions', 'x-nullable': true },
+    cook_minutes: { type: :integer, example: 666, 'x-nullable': true },
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] Style/TrailingCommaInHashLiteral: Avoid comma after the last item of a hash.